generated/broker_1/roster_3.json

```diff
@@ -372,19 +368,19 @@
           "start_on": "2016-08-01",
           "health": {
             "status": "Enrolled",
-            "employer_contribution": 8111.09,
-            "employee_cost": 4294.53,
-            "total_premium": 12405.62,
+            "employer_contribution": 0.0,
```

Comment:
Fixed it. 

https://github.com/dchealthlink/HBX-mobile-app-APIs/commit/5237dcc21fc260be41adf9458a634cdd900cc97d